### PR TITLE
Add Testimonial Spec

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Add follow-up spec with Congratulations message
* Add spec detailing testimonial link and testimonial videos

[#117070243]

### CODE REVIEW

File: js/app/app.js [hunk 1]
```diff
@@ -939,8 +939,11 @@ app.actions.sendEmail = function(data){
 	  type: "POST",
```
Comment (ericcf) [inline js/app/app.js]: Have you tested this email service to ensure that it works?

Comment (michaelwehrley) [inline js/app/app.js]: Absolutely not. 🐎  Not sure yet how to test it and likely a different story.  I came across this error code when I was testing, but I could remove it.

Comment (michaelwehrley) [inline js/app/app.js]: @ericcf I am moving these changes to another branch.

**Comment by ericcf** [comment]
🎋

**Pull request merged**